src/Api/Response/Response.php

```diff
@@ -90,4 +101,102 @@ public function getResultsCount()
 
         return $this->data[self::DATA_RESULTS]['count'];
     }
-}
\ No newline at end of file
+
+    public function getQueryInfoMessage(): string
+    {
+        if (!isset($this->data[self::DATA_QUERY_INFO_MESSAGE])) {
```

Comment:
Is this a scenario? If so, when can it happen and how can it be reproduced?

Reply:
Used a wrong function call. Fixed to check for empty array. 
There is a case when parsing DATA_QUERY_INFO_MESSAGE, that it can be an empty array, if no query data present in response.